acapy_wallet_upgrade/strategies.py

```diff
@@ -667,12 +681,14 @@ def __init__(
         self,
         uri: str,
         wallet_keys: Dict[str, str],
+        limit: int = 50,
         allow_missing_wallet: Optional[bool] = False,
         delete_indy_wallets: Optional[bool] = False,
         skip_confirmation: Optional[bool] = False,
     ):
         self.uri = uri
         self.wallet_keys = wallet_keys
+        self.limit = limit
```

Comment:
Oh, it already is in `self.limit` :slightly_smiling_face: In that case, I think we should just use it directly from the methods called from update items instead of passing it in.